### PR TITLE
Change doc to reflect new behaviour on schema change

### PR DESCRIPTION
**Pull request by rcillo**
Linked to https://github.com/zalando/nakadi/pull/1226

### CODE REVIEW

File: docs/_documentation/using_event-types.md [hunk 1]
```diff
@@ -143,13 +143,18 @@ reason not to provide any compatibility guarantee.
...
 #### Changing compatibility modes
 
-It's possible to change the compatibility mode from `none` to
-`forward` and from `forward` to `compatible`, e.g. it's possible to
-make the schema validation more strict but never more relaxed.
-
-It's not possible to upgrade directly from `none` to
-`compatible`. It's necessary to go first through `forward` for later
-upgrading to `compatible`.
+Compatibility modes are designed to offer data consumers with constraints
```
Comment (lmontrieux) [inline docs/_documentation/using_event-types.md:146]: `designed to offer data consumers with constraints on what can` -> `design to provide data consumers with guarantees about what can`

**Comment by lmontrieux** [comment]
I'm done with my review, just one small change

File: docs/_documentation/using_event-types.md (hunk 1 above)
Comment (rcillo) [reply docs/_documentation/using_event-types.md]: 👍 https://github.com/zalando/nakadi/pull/1227/commits/9985c98d33344327d44c16c0eecea338866ef0bc

**Comment by rcillo** [comment]
👍

**Comment by lmontrieux** [comment]
👍

**Pull request merged**